extensions/big-data-cluster/src/bigDataCluster/constants.ts

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import * as vscode from 'vscode';
+
+export enum BdcItemType {
+	controllerRoot = 'bigDataClusters.itemType.ControllerRootNode',
```

Comment:
Minor: casing is inconsistent here. `Endpoint` is TitleCased, all others are camelCased.
- Standard for enums is TitleCase, ideally should use this

Reply:
fixed.